README.md

```diff
@@ -200,6 +217,19 @@ Arguments:
 Note:
 - this link will be encrypted to the group you're linking to (i.e. link will have `recps: [groupId]`)
 
+### `ssb.tribes.link.createSubgroupLink({ group, subgroup, kaitiaki }, cb)`
+
+Creates a message of tyoe `link/group-group` which links a group to a subgroup
+
+Arguments:
+
+- `group` *GroupId* - the id of the parent private group
+- `subgroup` *GroupId* - the id of the subgroup you're linking to `group`
+- `kaitiaki` *Boolean* - a boolean to flag whether the subgroup is a **kaitiaki-only** subgroup
```

Comment:
do we need this yet?

Reply:
I think the kaitiaki concept is something which comes later.
"kaitiaki" is tricky as a word too because what they are kaitiaki of is totally context dependent (e.g. is it like admins of the subgroup.... or is this a group for the admins of the parent group?).

I have a hunch we should maybe store that logic on the ahau side when using these methods

Reply:
Hmm, yeah good point!